### PR TITLE
week 1 - `Pick`, `Readonly`

### PR DESCRIPTION
**Pull request by jeonghye-choi**
## week 1
- pick 
- readonly

### CODE REVIEW

File: jeonghye-choi/00004-easy-pick/template.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+type MyPick<T, K extends keyof T = keyof T> = {
```
Comment (jinyoung234) [inline jeonghye-choi/00004-easy-pick/template.ts:1]: K extends keyof T = keyof T << 이게 결국 K가 keyof T라는 뜻인건가요 ?!

**Review by jinyoung234 — APPROVED** [review]
LGTM :)!

File: jeonghye-choi/00004-easy-pick/readme.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  description: string;
+  completed: boolean;
+}
+```
+
+### Reference
+
+- https://www.typescriptlang.org/docs/handbook/2/generics.html#generic-constraints
+- https://www.typescriptlang.org/docs/handbook/2/keyof-types.html
+- https://www.typescriptlang.org/docs/handbook/2/mapped-types.html
```
Comment (Hyun-Jiii) [inline jeonghye-choi/00004-easy-pick/readme.md:34]: 👍

**Review by Hyun-Jiii — APPROVED** [review]
approve

**Pull request merged**